docs/source/GetStartedGuide.rst

```diff
@@ -211,6 +211,75 @@ keep reading.
           positional arguments, therefore test names (in case of
           ``avocado run``)
 
+Test References
+---------------
```

Comment:
Well this is quite detailed explanation, I'd not bother someone starting with Avocado with it. What would you think about moving this to `ReferenceGuide` section?